README.md

```diff
@@ -1 +1,39 @@
-# image-builder
\ No newline at end of file
+# Image Builder
+Image builder is an application to build docker image for running assignments. The image is built using user specific configurations.
+
+## Configurations
+There are two types of configurations provided - Code runner and Assignment Environment.
+
+### Code Runner Configurations
+- These configurations are required for running the code-runner server.
+- Code runner configurations are specified in yaml format and include instructions that are needed in Dockerfile for running the server.
+Following is a sample of the configuration yaml
+```commandline
+stages:
```

Comment:
What are the stages?
I feel "stages" doesn't make sense here. Why have another level of indirection when you can just have the top level as `serverConfig`?

Reply:
If it's going to be too much of a change, then we can take it up in another PR. Just make a note of it though.

Reply:
I had kept stages, just to keep my template structure generic. If at all there are multiple stages required in the dockerfile, then my template structure should be able to easily adopt it by simply adding another stage data into yaml.

Reply:
But as per your suggestion, I will remove the `stages` and do the rest of required changes in next PR.